tests/integration/common/s2n_test_scenario.py

```diff
@@ -239,6 +239,7 @@ def run_scenarios(test_func, scenarios):
     # get results, applying a 5 seconds limit for each task
     results.update((k, v.get(5000)) for k,v in results.items())
 
+    fail = 0
```

Comment:
Instead of initializing `fail`, shouldn't you change the `fail` on line 248 to `failed`?